### PR TITLE
Update kubeadm guide to use official stable deb and rpm repos

### PR DESCRIPTION
**Pull request by lukemarsden**
Thanks @mikedanese!

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1321)

### CODE REVIEW

**Comment by lukemarsden** [comment]
Still waiting on confirmation from tests that these instructions work. DO NOT MERGE YET.

File: docs/getting-started-guides/kubeadm.md [hunk 1]
```diff
@@ -55,26 +55,34 @@ For each host in turn:
 * SSH into the machine and become `root` if you are not already (for example, run `sudo su -`).
 * If the machine is running Ubuntu 16.04, run:
 
-      # apt-get install -y docker.io socat apt-transport-https
-      # curl -s -L \
-        https://storage.googleapis.com/kubeadm/kubernetes-xenial-preview-bundle.txz | tar xJv
-      # dpkg -i kubernetes-xenial-preview-bundle/*.deb
+      # curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+      # cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
```
Comment (mikedanese) [inline docs/getting-started-guides/kubeadm.md]: When to we use # vs $? isn't $ the traditional bash prompt?

Comment (lukemarsden) [inline docs/getting-started-guides/kubeadm.md]: I'm using `#` to indicate a root shell.

**Comment by lukemarsden** [comment]
We are good to go:
```
--> distro: centos7, docker: distro, multinode: 0, log: centos7-distro-0-1474988258.log
result: PASS
Docker  Version:         1.10.3
--> distro: xenial, docker: distro, multinode: 0, log: xenial-distro-0-1474987483.log
result: PASS
Docker  Version:      1.11.2
```

**Pull request merged**